README.md

```diff
@@ -0,0 +1,138 @@
+# py-ipfs-api
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+![](https://img.shields.io/pypi/v/ipfs-api.svg?style=flat-square)
+[![](https://img.shields.io/travis/ipfs/py-ipfs-api.svg?style=flat-square)](https://travis-ci.org/ipfs/py-ipfs-api)
+
+> A python client library for the IPFS API
+
+Check out [ipfs](http://ipfs.io/) and [the api command reference](http://ipfs.io/docs/commands/) for more information about the IPFS Api.
```

Comment:
Talking about standardization: Please always write *IPFS API* instead of *IPFS Api*.